ui/components/config/core-settings-list.tsx

```
@@ -167,6 +208,23 @@ export default function CoreSettingsList() {
 					/>
 				</div>
 
+				<div className="flex items-center justify-between space-x-2 rounded-lg border p-4">
+					<div className="space-y-0.5">
+						<label className="text-sm font-medium">Log Queue Size</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)